libs/dal/src/dal.service.ts

```diff
@@ -7,7 +7,7 @@ export class DalService {
   async connect(url: string, config: ConnectOptions = {}) {
     const baseConfig: ConnectOptions = {
       maxPoolSize: process.env.MONGO_MAX_POOL_SIZE || 500,
-      minPoolSize: process.env.NODE_ENV === 'production' ? 200 : 10,
+      minPoolSize: process.env.MONGO_MIN_POOL_SIZE || 10,
```

Comment:
With this change we need to add:
`MONGO_MIN_POOL_SIZE = 200`

to the following environment variable files to guarantee retrocompatibility for our users:
- `/apps/api/src/.env.production`
- `/apps/webhook/src/.env.production`
- `/apps/worker/src/.env.production`
- `/apps/ws/src/.env.production`